testsuite/MDAnalysisTests/test_atomgroup.py

```diff
@@ -2121,11 +2133,13 @@ def _check_universe(self, u):
         assert_equal(len(u.atoms[3].bonds), 2)
         assert_equal(len(u.atoms[4].bonds), 1)
         assert_equal(len(u.atoms[5].bonds), 1)
+        assert_('guess_bonds' in u._kwargs)
```

Comment:
The previous tests with a nonexistent kwarg. This is to test that we hold on to kwargs even after init is finished, which should be guaranteed since we do a deep copy of the kwarg dict. This was a convenient place to slot in a check on that.